lib/extensions/lettering.py

```diff
@@ -52,12 +45,19 @@ def __init__(self, *args, **kwargs):
 
         # font
         self.font_selector_box = wx.StaticBox(self, wx.ID_ANY, label=_("Font"))
-        self.update_font_list()
 
         self.font_chooser = wx.adv.BitmapComboBox(self, wx.ID_ANY, style=wx.CB_READONLY | wx.CB_SORT)
-        self.set_font_list()
         self.font_chooser.Bind(wx.EVT_COMBOBOX, self.on_font_changed)
 
+        self.font_filter = fs.FloatSpin(self, min_val=0, max_val=None, increment=1, value="0")
+        self.font_filter.SetFormat("%f")
+        self.font_filter.SetDigits(2)
+        self.font_filter.Bind(fs.EVT_FLOATSPIN, self.on_filter_changed)
+        self.font_filter.SetToolTip(_("Font size filter (mm)"))
```

Comment:
It would be clearer as "Font size filter (mm), 0 for all sizes."

Reply:
Yes, right. Thanks!